internal/saucecloud/playwright.go

```diff
@@ -59,11 +61,34 @@ func (r *PlaywrightRunner) RunProject() (int, error) {
 	}
 
 	passed := r.runSuites(app, otherApps)
-	if passed {
-		exitCode = 0
+	if !passed {
+		return 1, nil
 	}
 
-	return exitCode, nil
+	return 0, nil
+}
+
+func (r *PlaywrightRunner) setNodeRuntime(m framework.Metadata) error {
+	if !m.SupportGlobalNode() {
+		r.Project.NodeVersion = ""
+		return nil
+	}
+	if r.Project.NodeVersion == "" {
+		return nil
+	}
```

Comment:
@mhan83 Should we already have the defaulting to the "default" runtime version here? Or give users a chance to define it for themselves first, before we introduce defaulting with a patch later? Right now, I'm not sure how many people would actually bother setting the node version explicitly, since we only support _one version_. Later this year, Node.js 22 will reach LTS, which is when things get interesting! The earlier we get the defaulting logic out, the faster we can hope to complete the migration. I'd vote for introducing defaulting from the get go.

Reply:
I *think* I'm in favor of defaulting from the start...one question I have though is do we want a way for users to explicitly use the bundled node binary? If we default from the get go and there is some edge case that gets exposed, it might be easier for the user to opt-out instead of rolling back their saucectl version. This would probably slow the migration though...